sp800-63b/sec1_2_introduction.md

```diff
@@ -4,29 +4,29 @@
 
 _This section is informative._
 
-This document and its companion documents, [[Special Publication (SP) 800-63-3]](sp800-63-3.html), [[SP 800-63A]](sp800-63a.html), and [[SP 800-63C]](sp800-63c.html), provide technical guidelines to agencies for the implementation of digital authentication.
+This document and its companion documents, [Special Publication (SP) 800-63](sp800-63-3.html), [SP 800-63A](sp800-63a.html), and [SP 800-63C](sp800-63c.html), provide technical guidelines to agencies for the implementation of digital authentication.
```

Comment:
We have been inconsistent in the naming of section reference anchors -- sometimes naming them by topic like "throttle" and sometimes by section number -- and of course the former works better as the documents are revised. But these documents also want to be usable in printed form, and we don't have much option other than to reference section numbers. Perhaps we just need to consider section numbers frozen (like they do with controls in -53) and just mark any removed sections as "deleted" or "moved".